internal/c/libqb.cpp

```diff
@@ -27355,51 +27355,23 @@ void sub__icon(int32 handle_icon, int32 handle_window_icon, int32 passed) {
 #endif // DEPENDENCY_ICON
 
 int32 func_screenwidth() {
-#ifdef QB64_GLUT
-    //while (!window_exists) {
-    //    Sleep(100);               //endless loop which is completely unneeded here.
-    //}                             //we can get the desktop's width and height without there needing to be an active screen (such as $SCREENHIDE)
-
-#    ifdef QB64_WINDOWS
-
-    int32 waited_enough = 20;
-
-    while (!window_handle&&waited_enough>0) {
-        Sleep(100);
-        waited_enough = waited_enough - 1;
-    }
-    if (waited_enough == 0) return 0; //if the screen isn't created and doesn't have a handle after two seconds, then return a value of 0.  Don't run an endless loop!
-#    endif
-    return glutGet(GLUT_SCREEN_WIDTH);
-#else
-#    ifdef QB64_WINDOWS
+#ifdef QB64_WINDOWS
     return GetSystemMetrics(0);

```

Comment:
It's a small thing, but there are constants you're intended to use for `GetSystemMetrics()` rather than hard-coded values. You should try swapping `0` for `SM_CXSCREEN` and `1` for `SM_CYSCREEN`, it makes it much easier to know what's going on. Hopefully those constants are already available in this file via the `#include`s, if they can't be found then I'm fine leaving it as-is :)